sample_apps/run_example.sh

```diff
@@ -3211,9 +3211,14 @@ function run_test-crypto_algorithms() {
     # public-key algo 'ecc-384' is failing ... Comment out for now ...
     # for public_key_algo in rsa-3072 rsa-4096 ecc-384; do
     for public_key_algo in rsa-3072 rsa-4096; do
-        # for symmetric_key_algo in aes-256-cbc-hmac-sha256; do
-        symmetric_key_algo="aes-256-cbc-hmac-sha256"
-
+        # RESOLVE: Investigate & fix these limitations:
+        #  - generate_symmetric_key() does not support
+        #    Enc_method_aes_128_cbc_hmac_sha256; i.e., "aes-128-cbc-hmac-sha256";
+        # - unprotect_blob() only allows Enc_method_aes_256_cbc_hmac_sha256
+        # - These two should also be supported in loop below:
+        #   - aes-256-cbc-hmac-sha384, aes-256-gcm
```

Comment:
@jlmucb -- We need to discuss whether the limitations in the fns-mentioned are valid. 

The 3 authenticated symmetric-key algorithms named above do not work when we attempt to run `simple_app` with the newly added  `--auth_symmetric_key_alg` flag naming these algorithms.